pkg/ccl/schemachangerccl/testdata/explain/drop_table_multiregion

```diff
@@ -48,9 +49,10 @@ Schema change plan for DROP TABLE ‹multi_region_test_db›.‹public›.‹tab
  │         │    ├── VALIDATED   → ABSENT  PrimaryIndex:{DescID: 108, IndexID: 1, ConstraintID: 1}
  │         │    ├── PUBLIC      → ABSENT  IndexPartitioning:{DescID: 108, IndexID: 1}
  │         │    └── PUBLIC      → ABSENT  IndexName:{DescID: 108, Name: table_regional_by_row_pkey, IndexID: 1}
- │         └── 15 Mutation operations
+ │         └── 16 Mutation operations
  │              ├── MarkDescriptorAsDropped {"DescriptorID":108}
  │              ├── RemoveAllTableComments {"TableID":108}
+ │              ├── NotImplemented {"ElementType":"scpb.TablePartit..."}
```

Comment:
Ruh roh. Looks like you need to add `TablePartitioning` to that set of elements which will be no-opped when dropping tables and the like. The op-rule in question is named `skip element removal ops on descriptor drop`.